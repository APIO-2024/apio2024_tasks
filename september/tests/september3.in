txy4h26c1rm1uv8tr3eonahd67u8h56x
4
7 1
0 0 1 3 4 5
6 2 5 4 3 1
10 1
0 0 2 3 4 4 6 7 8
5 9 1 8 7 6 3 4 2
7 1
0 1 1 3 4 4
2 6 5 4 3 1
6 1
0 1 2 2 3
3 5 4 1 2
