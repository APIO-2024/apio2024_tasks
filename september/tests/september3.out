7ckgnn4wyi495puj3ibqf81dqvapyv6b
6
8
6
3
